pkg/apis/v1alpha1/dnspolicy_types.go

```diff
@@ -34,20 +34,18 @@ type DNSPolicySpec struct {
 	// +optional
 	HealthCheck *HealthCheckSpec `json:"healthCheck,omitempty"`
 
-	// +kubebuilder:validation:Required
-	// +required
+	// +optional
 	LoadBalancing *LoadBalancingSpec `json:"loadBalancing"`
 }
 
 type LoadBalancingSpec struct {
-	// +required
+	// +optional
 	Weighted *LoadBalancingWeighted `json:"weighted,omitempty"`
```

Comment:
This is no longer mandatory, and the only thing required in the policy spec is the targetRef, minimum policy required now looks like:

```.yaml
apiVersion: kuadrant.io/v1alpha1
kind: DNSPolicy
metadata:
  name: prod-web
  namespace: multi-cluster-gateways
spec:
  targetRef:
    name: prod-web
    group: gateway.networking.k8s.io
    kind: Gateway
```

This came about as feedback during the demo, there really is no reason it needs to be required. If you want to change the default value (120), you can still change it by updating the spec but it's really meaningless unless you are applying custom weights.